### PR TITLE
apps/poll/userview: adding validation logic and max chars

### PR DESCRIPTION
**Pull request by khamui**

### CODE REVIEW

File: adhocracy4/polls/assets/CharCounter.jsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+export const CharCounter = (props) => {
+  const current = props.value.length
+
+  return (
+    <span>{current}/{props.max}</span>
+  )
+}
```
Comment (phillimorland) [inline adhocracy4/polls/assets/CharCounter.jsx:9]: super nice!

**Review by phillimorland — COMMENTED** [review]
very cool but I cannot see the char count, is this dues to a styling issue? if so go ahead and merge :)

File: adhocracy4/polls/assets/PollQuestion.jsx [hunk 2]
```diff
@@ -87,15 +105,26 @@ export const PollQuestion = (props) => {
...
-                      <input
-                        className="input-group__input"
-                        type="text"
-                        name="question"
-                        id={'id_choice-' + choice.id + '-single'}
-                        value={checked ? otherChoiceAnswer : ''}
-                        onChange={(event) => { handleOtherChange(event) }}
-                        disabled={!props.question.authenticated || !checked}
-                      />}
+                      <>
```
Comment (fuzzylogic2000) [inline adhocracy4/polls/assets/PollQuestion.jsx:108]: What are these empty tags for?

**Review by fuzzylogic2000 — APPROVED** [review]
Nice! I can see the count and trigger the error. And will merge now, even though I do not get the empty html tags.

**Pull request merged**